### PR TITLE
typo fix and more info about elf2uf2

### PR DESCRIPTION
**Pull request by Pavan-Parker**

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Choose a default "cargo run" tool.
 # probe-run is recommended if you have a debugger
-# elf2uf2-rs loads firmware over USB when the rp2040 is in boot mode
+# elf2uf2-rs loads firmware over USB when the rp2040 is in boot mode, which required you to install corresponding package by executing "$ cargo install elf2uf2-rs"
```
Comment (jonathanpallant) [inline .cargo/config.toml:3]: I don't think this comment is needed - both probe-run and elf2usb2-rs require installation.

File: .cargo/config.toml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
 runner = "probe-run --chip RP2040"
-# runner = "elf2uf2-rs -d
+# runner = "elf2uf2-rs -d"
```
Comment (jonathanpallant) [inline .cargo/config.toml:6]: But this is a good spot. Oops!

**Review by jonathanpallant — CHANGES_REQUESTED** [review]

File: .cargo/config.toml (hunk 1 above)
Comment (jonathanpallant) [reply .cargo/config.toml:3]: I've added this to the README in #14

**Comment by 9names** [comment]
These changes were also in #18 which had a few other fixes, so I've merged that one instead.
Thanks for the PR @Pavan-Parker

**Pull request closed without merging**